### DISCUSSION TITLE
How to make a multisite installation?

### DISCUSSION THREAD

**Discussion by ced64k** [datenstrom/community#453] [Ask a question] [answered]
Hello,

I would like to know if it's possible to use Yellow for multisite. For example site1.mysite.com and site2.mysite.com that would share the same installation and layouts, using conditions if I'm on site1 or site2, etc.

Thank you.

**Comment by schulle4u** [comment, marked as answer]
As far as I know this won't work out of the box. Assuming all subdomains belong to the same webserver user, you could try to install Yellow for the first domain and create symbolic links to the extensions and layouts in each subdomain. Just an early morning idea, I've never tried this. 😀

**Comment by markseuffert** [comment]
The general advice is to make two/multiple installations. Datenstrom Yellow is small. A basic installation doesn't take much hard disk space. For developers it shouldn't be too hard to add a multisite feature, the core is pretty flexible, check out the API and feel free to ask questions.

**Comment by ced64k** [comment]
Thank for the advices :)
